lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -58,7 +58,7 @@ def modx_url(self, problem_location, dispatch):
         problem_location: location of the problem on which we want some action
 
         dispatch: the the action string that gets passed to the view as a kwarg
-            example: 'check_problem' for having responses processed
+            example: 'submit_problem' for having responses processed
```

Comment:
The name of this event did not change-- please update the doc (revert the change).